src/views/RecIM/components/List/Listing/index.js

```diff
@@ -17,36 +27,41 @@ const ActivityListing = ({ activity }) => {
   let registrationStart = DateTime.fromISO(activity.RegistrationStart);
   let registrationEnd = DateTime.fromISO(activity.RegistrationEnd);
   return (
-    <ListItem button component={Link} to={`/recim/activity/${activity.ID}`} className="gc360_link">
-      <Grid container className={styles.listing}>
-        <Grid item xs={3}>
-          {activity.Name}
+    <ListItemButton component={Link} to={`/recim/activity/${activity.ID}`} className="gc360_link">
```

Comment:
This was an MUI5 change right? I remember seeing stuff around swapping this component. I see you did it for the other list items as well. Do they all still work as intended? Like linking correctly. Thanks for making this change BTW!